models/app.models.js

```diff
@@ -34,3 +34,19 @@ exports.selectCommentsByArticleId = (article_id) => {
     return rows;
   });
 };
+
+exports.addComment = (article_id, username, body) => {
+  const text = `INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING comment_id, article_id, author, body, created_at;`;
+
+  const values = [article_id, username, body];
+
+  return db
+    .query(text, values)
+    .then(({ rows }) => {
+      //console.log(rows, "<=rows");
+      return rows;
```

Comment:
as per my comment on your test if you take the object out of the array here before returning to the controller this is best practices - we are only sending one comment back for this endpoint so need for it to be in an array